src/tests/ftest/security/pool_security_acl.py

```diff
@@ -0,0 +1,401 @@
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+'''
+from __future__ import print_function
+
+import os
+from apricot import TestWithServers
+from daos_utils import DaosCommand
+from dmg_utils import DmgCommand
+import dmg_utils
+import random
+import pwd
+import grp
+import re
+from general_utils import pcmd
+
+
+PERMISSIONS = ["", "r", "w", "rw"]
+
+def acl_entry(usergroup, name, permission):
+    '''
+    Deascription:
+        Create a daos acl entry on the specified user or groups
+        with specified or random permission.
+    Args:
+        usergroup: user or group.
+        name: user or group name to be created.
+        permission: permission to be created.
+    Return:
+        entry: daos acl entry.
+    '''
+    if permission == "random":
+        permission = random.choice(PERMISSIONS)
+    if permission == "nonexist":
+        return ""
+    if "group" in usergroup:
+        entry = "A:G:" + name + "@:" + permission
+    else:
+        entry = "A::" + name + "@:" + permission
+    return entry
+
+def add_del_user(hosts, ba_cmd, user):
+    '''
+    Deascription:
+        Add or delete the daos user and group on host by sudo command.
+    Args:
+        hosts: list of host.
+        ba_cmd: linux bash command to create user or group.
+        user: user or group name to be created or cleaned.
+    Return:
+        none.
+    '''
+    bash_cmd = os.path.join("/usr/sbin", ba_cmd)
+    homedir = ""
+    if "user" in ba_cmd:
+        homedir = "-r"
+    cmd = " ".join(("sudo", bash_cmd, homedir, user))
+    pcmd(hosts, cmd, False)
+
+class PoolSecurityTest(TestWithServers):
+    '''
+    Test Class Description:
+    Tests to verify the Pool security with dmg and daos tools on daos_server.
+    :avocado: recursive
+    '''
+
+    def get_pool_acl_list(self, uuid):
+        '''
+        Deascription:
+            Get daos pool acl list by dmg get-acl.
+        Args:
+            uuid: pool uuid number.
+        Return:
+            pool_permission_list: daos pool acl list.
+        '''
+        dmg = DmgCommand(os.path.join(self.prefix, "bin"))
+        dmg.request.value = "pool"
+        dmg.action.value = "get-acl --pool " + uuid
+        port = self.params.get("port", "/run/server_config/*")
```

Comment:
Optional: I don't think you need to explicitly define this. You can directly use self.port defined in server_utils.py